StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewModel.swift

```diff
@@ -56,7 +93,9 @@ class UpdatePaymentMethodViewModel {
             fatalError("Unsupported payment type \(paymentMethod.type) in UpdatePaymentMethodViewModel")
         }
         self.paymentMethod = paymentMethod
-        self.appearance = appearance
+        var disabledAppearance = appearance
+        disabledAppearance.colors.componentBackground = disabledBackgroundColor(componentBackground:    disabledAppearance.colors.componentBackground)
+        self.appearance = disabledAppearance
```

Comment:
Do we not use this new disabled look all though PaymentSheet? The fields are also disable when we are in the processing state.

Reply:
I'm asking to check if we're supposed to use it throughout all of PaymentSheet. If so, then I'll redo this so that we have a translucent view overlaying the text fields and just set the background color of the overlay view. I didn't want to do that when I thought it was only for the update pm screen because that would affect the text color, which is why I did the calculations for the background color instead.

Reply:
Hannah said it should only apply to the text fields on the update pm screen. I'll move the color calculation to another file though